src/plugins/console/public/lib/mappings/mappings.ts

```diff
@@ -362,7 +412,12 @@ const retrieveTemplates = async (http, settingsToRetrieve, dataSourceId) => {
  * @param settings Settings A way to retrieve the current settings
  * @param settingsToRetrieve any
```

Comment:
This is the reason why I left settingsToRetrieve as ```any``` and didn't add typing for it.